gemini/annotations.py

```diff
@@ -264,6 +264,25 @@ def _get_var_coords(var, naming):
         chrom = _get_chr_as_grch37(chrom)
     return chrom, start, end
 
+def _get_var_ref_and_alt(var):
+    """Retrieve variant reference and alternate alleles from multiple input objects.
+    """
+    if isinstance(var, basestring):
+        # Assume var is a line from a VCF. 
+        ref, alt = var.split('\t')[3:5]
+    elif isinstance(var, dict) or isinstance(var, sqlite3.Row):
+        ref = var["ref"]
+        alt = var["alt"]
+    else:
+        try:
+            ref = var.REF
+            alt = var.ALT
+        except KeyError:
```

Comment:
ok. so I think this will always be a pysam object, and it will *always* give a keyerror, so better to put the var.ref, var.alt, first or only.
but not necessary for this PR, I was just trying to understand.